.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - nightly
```

Comment:
While that's true, we do not really require features which are only 5.6+. Thus, no harm in supporting 5.5; if we won't be able to anymore at some point, we can just drop it then.